devdocs/process.md

```diff
@@ -227,42 +232,84 @@ The meaning of issue prefixes: `s.` status, `a.` aspect, `f.` feature, `t.` type
 * `p.Low`: Very little impact, unlikely to do in the near future.
 * `p.Zero`: Unlikely to do, ever.
 
+**Category**
+
+* Changes to _functionality_, categorized based on size
+  * `c.Enhancement`: An enhancement to an existing functionality (not big enough 
+   consider as a story).
+  * `c.Story`: A user story.
+  * `c.Epic`: A feature that is worth many user stories.
```

Comment:
so, we would be creating one Epic which is linked to multiple Stories. Then when all Stories are done, we close the Epic. Any future modification to the feature would be an Enhancement.

Am I right to interpret that way?